IBX-3587: Segment management (#193)
ezsystems/user-documentation: docs/personalization/segment_management.md

Segments available in the **Elements** sections are reusable. It means, you can use the same segment in different segment groups.

![Parent segment group](img/perso_segment_group_and_parent.png "Parent segment group")

### Create segment group with AND logic condition

The following example shows segment groups with `AND` operators linking nested elements:

- women
- Poland 
- sales hunters (as a type of customers.)

![AND segment group logic](img/perso_segment_group_sales_hunters.png "AND segment group logic")

All three criteria are linked with `AND`operator so, all conditions must be fullfiled. 

Recommendation call in scenario using model with segments, contains requests to all specified segments with `AND` conditions:

`https://reco.test.perso.ibexa.co/api/v2/41307/588/landing_page?numrecs=6&attribute=ses_name,title,ses_image,teaser_image&crosscontenttype=1&segments=7,11,14`

Where segments ID correspond to segment groups:

- women - `segment ID=7`
- Poland - `segment ID=11`
- sales hunters - `segment ID=14`

As a result, a reco call returns two events which qualify for these segment groups requirements.

> Two items with `ID=587` and `ID=588` relevant for the following segment combinations, clicked by:

Reviewer: ```suggestion
Two items with `ID=587` and `ID=588` are relevant for the following segment combinations, clicked by:
```

                  rendered: "//event.test.perso.ibexa.co/api/41307/rendered/someuser/57/588?scenario=landing_page&modelid=10316421&categorypath=&requestuuid=276a5930-dea3-11ed-8cdf-92a64ae30943"
               },
               attributes: [
                  {
                     key: "title",
                     values: [
                        "Minimalist luxury in a small and stylish bedroom"
                     ]
                  },
                  {
                     key: "teaser_image",
                     values: [
                        "/var/site/storage/images/0/0/7/0/700-1-eng-GB/de8a98767362-bedroom2.jpg"
                     ]
                  }
               ]
            }
         ]
      }
      ```

### Create segment group with OR logic condition

The following example shows segment groups with `OR` operator connecting nested elements:

- women
- Poland
- 25-35 (age)
- Germany

![OR segment group logic](img/perso_segment_group_or.png "OR segment group logic")

> In these case to get recommendations, only one condition must be met: women from Poland or women from Germany.

Reviewer: ```suggestion
In this case to get recommendations, only one condition must be met: women from Poland or women from Germany.
```

`https://reco.test.perso.ibexa.co/api/v2/41307/someuser/landing_page?numrecs=6&attribute=ses_name,title,ses_image,teaser_image&crosscontenttype=1&segments=7,8,10,11`

Where segments ID correspond to segment groups:

- women - `segment ID=7`
- Poland - `segment ID=11`
- 25-35 - `segment ID=8`
- Germany - `segment ID=10`

> As a result, a reco call returns only one event which qualify for these segment group requirements. The item with `ID=587` relevant for this segment combinations, clicked by:

Reviewer: ```suggestion
As a result, a reco call returns only one event which qualifies for these segment group requirements. The item with `ID=587` is relevant for this segment combination, clicked by:
```
